storage/storage.go

```diff
@@ -19,15 +19,11 @@ type Storage interface {
 	AddPiece(ctx context.Context, sector abi.SectorID, pieceSizes []abi.UnpaddedPieceSize, newPieceSize abi.UnpaddedPieceSize, pieceData Data) (abi.PieceInfo, error)
 }
 
-type FallbackPostOut struct {
-	PoStInputs []PoStCandidateWithTicket
-	Proof      []abi.PoStProof
-}
-
 type Prover interface {
-	GenerateEPostCandidates(ctx context.Context, miner abi.ActorID, sectorInfo []abi.SectorInfo, challengeSeed abi.PoStRandomness, faults []abi.SectorNumber) ([]PoStCandidateWithTicket, error)
-	GenerateFallbackPoSt(ctx context.Context, miner abi.ActorID, sectorInfo []abi.SectorInfo, challengeSeed abi.PoStRandomness, faults []abi.SectorNumber) (FallbackPostOut, error)
-	ComputeElectionPoSt(ctx context.Context, miner abi.ActorID, sectorInfo []abi.SectorInfo, challengeSeed abi.PoStRandomness, winners []abi.PoStCandidate) ([]abi.PoStProof, error)
+	GenerateWinningPoStSectorChallenge(proofType abi.RegisteredProof, minerID abi.ActorID, randomness abi.PoStRandomness, eligibleSectorCount uint64) ([]uint64, error)
```

Comment:
@magik6k - What does each `uint64` in the returned slice represent? Indices in proving set? sector numbers?

Reply:
Can we newtype or alias those uint64 going out?

Reply:
This as far I understand is the sector index in the proving partition AMT which doesn't exist in specs-actors yet.

I expect the type to be defined in specs-actors when windowed post is fully implemented in there